### PR TITLE
[DEVX-1806] Update doc and tests for cypress 10+

### PR DESCRIPTION
**Pull request by tianfeng92**
The plugin works for cypress10+ but requires different way to config it. The `plugin` folder has been deprecated from cypress. It should be configured in cypress config file instead.

### CODE REVIEW

File: tests/integration/cypress.spec.js [hunk 1]
```diff
@@ -63,23 +61,4 @@ describe('runs tests on cloud', function () {
     expect(assets['video.mp4']).toBe('video.mp4');
     expect(assets.video).toBe('video.mp4');
   });
-
-  test('job has name/tags correctly set', async function () {
```
Comment (mhan83) [inline tests/integration/cypress.spec.js:67]: Why is this test removed?

File: tests/integration/tsconfig.json [hunk 2]
```diff
@@ -0,0 +1,3 @@
+{
```
Comment (mhan83) [inline tests/integration/tsconfig.json:1]: I don't see an ts in the integration dir, so why is this tsconfig needed?

**Review by mhan83 — COMMENTED** [review]
Looks good, but I just have some general thoughts about how the README should be structured. imo its nice when instructions are presented in a way where its clear what the minimum setup is and what the optional setup is. For this plugin, the minimum is:
1. Install plugin
2. Setup sauce creds
3. Plugin registration via `cypress.config.*` or `cypress/plugins/index.js`
4. Run test 
5. 🎉 

Optional:
1. Adding metadata (e.g. build, tags, region).

File: tests/integration/cypress.spec.js (hunk 1 above)
Comment (tianfeng92) [reply tests/integration/cypress.spec.js:67]: added another way to test the build and tags

**Review by alexplischke — APPROVED** [review]

**Review by mhan83 — APPROVED** [review]

**Pull request merged**